pandas/core/window.py

```diff
@@ -312,6 +312,9 @@ class Window(_Window):
     * ``gaussian`` (needs std)
     * ``general_gaussian`` (needs power, width)
     * ``slepian`` (needs width).
+
+    To learn more about the frequency strings, please see `this link
+    <http://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases>`__.
```

Comment:
Since `freq` keyword is deprecated, should I remove the `learn more about..` text from all places where the only reference to `frequency strings` is in `freq` parameter?